pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -54,6 +55,9 @@ const (
 	SCSIControllerDeviceLimit = 15
 	SCSIDeviceSlots           = 16
 	SCSIReservedSlot          = 7
+	DefaultAdapterType        = "lsiLogic"
```

Comment:
It is used in CreateVolume for creating disk (Line# 920), which requires camel case. "lsilogic" const (all lower case) results in error "Invalid Argument". Yet, I changed LSILogicSASControllerType to camel case and used string.ToLower method wherever lower case is applied.